tests/Doctrine/TypeFunctionalTestCase.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Brick\Geo\Tests\Doctrine;
+
+use Brick\Geo\Tests\Doctrine\Fixtures;
+use Doctrine\Common\DataFixtures\Executor\ORMExecutor;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\DataFixtures\Loader;
+use Doctrine\Common\DataFixtures\Purger\ORMPurger;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Tools\SchemaTool;
+use Doctrine\ORM\Tools\Setup;
+use Doctrine\Tests\DBAL\Mocks\MockPlatform;
+use Doctrine\Tests\DbalFunctionalTestCase;
+
+class TypeFunctionalTestCase extends DbalFunctionalTestCase
+{
+
+    /**
+     * @var MockPlatform
+     */
+    private $platform;
+
+    /**
+     * @var Loader
+     */
+    private $fixtureLoader;
+
+    /**
+     * @var SchemaTool
+     */
+    private $schemaTool;
+
+    /**
+     * @var EntityManager
+     */
+    private $em;
+
+    /**
+     * @var ORMExecutor
+     */
+    private $ormExecutor;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        if (getenv('ENGINE') == 'SQLite3' || getenv('ENGINE') == 'GEOS') {
+            $this->markTestSkipped('The doctrine types currently only work with MySQL and PostgreSQL');
```

Comment:
AFAIK Doctrine ORM also works with SQLite, would you mind adding tests for SQLite as well?